.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,7 +1,6 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[Bug] Enter issue title here"
```

Comment:
I wonder could just keeping [Bug] be better?
This way it can be easily seen in the list? 
Maybe I misunderstand what title means in this context?

Reply:
My thinking was that we have tags to show something is a bug, so the [Bug] is like a tag that is sometimes, but not always, there as well. I focus on tags on this repo, because they're consistent and people tend to not mess with them as much as with titles :)

Reply:
![image](https://github.com/BeamMP/BeamMP-Server/assets/29932116/db862fe5-db1d-4715-b37a-037aac323c68)

and this works better than visually filtering for me, and i dont consider bugs more or less important than features

Reply:
Fair enough then :)

Will approve now.